content/v2/domains/dnssec.markdown

```diff
@@ -0,0 +1,228 @@
+---
+title: DNSSEC API | Domains | DNSimple API v2
+excerpt: This page documents the DNSimple DNSSEC API v2.
+---
+
+# DNSSEC API
+
+* TOC
+{:toc}
+
+
```

Comment:
We should add the "beta" banner. I'd also add a short intro that explains what DNSSEC is for, when you should use DNSSEC API vs DS Record API.

Nothing too complex, but to avoid devs would start sending DS record requests when they just need to use enable/disable DNSSEC.